api/secrets/secret_handler.go

```diff
@@ -0,0 +1,616 @@
+		)
+	}
+	return secrets, nil
+}
+
+func (eh SecretHandler) getBlobFuseSecrets(component v1.RadixCommonDeployComponent, envNamespace string, volumeMount v1.RadixVolumeMount) (models.Secret, models.Secret) {
+	return eh.getAzureVolumeMountSecrets(envNamespace, component,
+		defaults.GetBlobFuseCredsSecretName(component.GetName(), volumeMount.Name),
+		volumeMount.Name,
+		defaults.BlobFuseCredsAccountNamePart,
+		defaults.BlobFuseCredsAccountKeyPart,
+		defaults.BlobFuseCredsAccountNamePartSuffix,
+		defaults.BlobFuseCredsAccountKeyPartSuffix,
+		models.SecretTypeAzureBlobFuseVolume)
+}
+
+func (eh SecretHandler) getCsiAzureSecrets(component v1.RadixCommonDeployComponent, envNamespace string, volumeMount v1.RadixVolumeMount) (models.Secret, models.Secret) {
+	return eh.getAzureVolumeMountSecrets(envNamespace, component,
+		defaults.GetCsiAzureCredsSecretName(component.GetName(), volumeMount.Name),
+		volumeMount.Name,
+		defaults.CsiAzureCredsAccountNamePart,
+		defaults.CsiAzureCredsAccountKeyPart,
+		defaults.CsiAzureCredsAccountNamePartSuffix,
+		defaults.CsiAzureCredsAccountKeyPartSuffix,
+		models.SecretTypeCsiAzureBlobVolume)
+}
+
+func (eh SecretHandler) getAzureVolumeMountSecrets(envNamespace string, component v1.RadixCommonDeployComponent, secretName, volumeMountName, accountNamePart, accountKeyPart, accountNamePartSuffix, accountKeyPartSuffix string, secretType models.SecretType) (models.Secret, models.Secret) {
+	accountkeyStatus := models.Consistent.String()
+	accountnameStatus := models.Consistent.String()
+
+	secretValue, err := eh.client.CoreV1().Secrets(envNamespace).Get(context.TODO(), secretName, metav1.GetOptions{})
+	if err != nil {
+		log.Warnf("Error on retrieving secret '%s'. Message: %s", secretName, err.Error())
+		accountkeyStatus = models.Pending.String()
+		accountnameStatus = models.Pending.String()
+	} else {
+		accountkeyValue := strings.TrimSpace(string(secretValue.Data[accountKeyPart]))
+		if strings.EqualFold(accountkeyValue, secretDefaultData) {
+			accountkeyStatus = models.Pending.String()
+		}
+		accountnameValue := strings.TrimSpace(string(secretValue.Data[accountNamePart]))
+		if strings.EqualFold(accountnameValue, secretDefaultData) {
+			accountnameStatus = models.Pending.String()
+		}
+	}
+	//"accountkey"
+	accountKeySecretDTO := models.Secret{Name: secretName + accountKeyPartSuffix, DisplayName: fmt.Sprintf("Account Key"), Resource: volumeMountName, Component: component.GetName(), Status: accountkeyStatus, Type: secretType}
+	//"accountname"
+	accountNameSecretDTO := models.Secret{Name: secretName + accountNamePartSuffix, DisplayName: fmt.Sprintf("Account Name"), Resource: volumeMountName, Component: component.GetName(), Status: accountnameStatus, Type: secretType}
+	return accountKeySecretDTO, accountNameSecretDTO
+}
+
+//func (eh SecretHandler) getAzureKeyVaultSecret(component v1.RadixCommonDeployComponent, envNamespace, secretName, keyVaultNamePart string) (models.Secret, models.Secret) {
```

Comment:
Cleanup commented code

Reply:
It is replaced by `getCredentialSecretsForSecretRefs`. Removed